robotoff/app/core.py

```diff
@@ -332,24 +359,25 @@ def save_annotation(
                 tx.rollback()
                 raise e
 
-        # If the top annotation has more than 2 votes, consider applying it to the insight.
-        if existing_votes[0].num_votes > 2:
-            annotation = existing_votes[0].value
-            verified = True
-
-        # But first check for the following cases:
-        #  1) The 1st place annotation has >2 votes, and the 2nd place annotation has >= 2 votes.
-        #  2) 1st place and 2nd place have 2 votes each.
-        #
-        # In both cases, we consider this an ambiguous result and mark it with 'I don't know'.
-        if (
-            existing_votes[0].num_votes >= 2
-            and len(existing_votes) > 1
-            and existing_votes[1].num_votes >= 2
-        ):
-            # This code credits the last person to contribute a vote with a potentially not their annotation.
-            annotation = 0
-            verified = True
+        if existing_votes:
+            # If the top annotation has more than 2 votes, consider applying it to the insight.
+            if existing_votes[0].num_votes > 2:
```

Comment:
I think it would be more clear to use "3 votes or more" 
As I read the code, the "strictly more" was not obvious, and I was asking myself, how could the second case arise.
```suggestion
            # If the top annotation has 3 or more votes, consider applying it to the insight.
            if existing_votes[0].num_votes >= 3:
```